src/components/MapPage/Map.js

```diff
@@ -1,34 +1,229 @@
-import React from 'react';
-import { withScriptjs, withGoogleMap, GoogleMap } from 'react-google-maps';
-import { MarkerClusterer } from 'react-google-maps/lib/components/addons/MarkerClusterer';
-import OrganizationMarker from './OrganizationMarker';
-import { compose, lifecycle } from 'recompose';
+import React from "react";
+import { withScriptjs, withGoogleMap, GoogleMap } from "react-google-maps";
+import { MarkerClusterer } from "react-google-maps/lib/components/addons/MarkerClusterer";
+import OrganizationMarker from "./OrganizationMarker";
+import { compose, lifecycle } from "recompose";
+
+const styles = [
```

Comment:
this styles is a little big. You might want to break it off into its own file and import in

GoogleMapConfig.js
```
export const googleMapConfig = [...]

```

Map.js
```
import {googleMapConfig} from './GoogleMapConfig'

// ...

<GoogleMap
...
defaultOptions={googleMapConfig}

```

Reply:
agree